css/style.css

```diff
@@ -0,0 +1,149 @@
+html {
+  /* Setting a base font size of 10px give us easier rem calculations
+         Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+       */
+  font-size: 10px;
+  margin: 0;
+}
+
+body {
+  /* Native font stack https://getbootstrap.com/docs/4.2/content/reboot/#native-font-stack */
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
+    "Helvetica Neue", Arial, sans-serif;
+  font-size: 2rem;
+  line-height: 1.5;
+  color: #477ca7;
+  background: #333;
+  margin: 0px;
+}
+
+#main-doc {
+  display: block;
+  margin: auto;
+  border-radius: 5px;
+  background: #333;
+}
+
+#navbar {
+  width: 25%;
+  margin: 5px;
+  float: left;
+  position: fixed;
+}
+
+#navbar h1 {
+  text-align: center;
+}
+#navbar ul {
+  list-style: none;
+  padding-left: 10px;
+}
+#navbar li {
+  line-height: 2;
+  font-size: 2rem;
+  color: whitesmoke;
+}
+
+#navbar li li:hover {
+  transform: scale(1.1);
+}
+
+@media (max-width: 460px) {
+  #navbar {
+    font-size: 1rem;
+    line-height: 1;
+    float: clear;
+  }
+}
+
+#content-wrapper {
+  padding-top: 50px;
+  display: block;
+  max-width: 70%;
+  float: right;
+  padding-right: 15px;
+  background-color: #477ca7;
+  color: whitesmoke;
+  opacity: 0.9;
+  padding-left: 15px;
+}
+
+@media (max-width: 460px) {
+  #content-wrapper {
+    font-size: 1rem;
+    line-height: 1;
+    float: clear;
+  }
+}
+
```

Comment:
I guess this is a duplicate of your css rules from line 51 - 57 of this same file. So, you should remove either of the two.

Reply:
Great job @belmeetmule 🤝

- [ ] [Optional] Well explained comments. A little politeness could do wonders. ✔️As stated earlier by the previous reviewer, you may please try to be more human-friendly by using emojis.